_targets.R

```diff
@@ -1,15 +1,21 @@
 library(targets)
 library(tibble)
 library(tarchetypes)
+library(dplyr)
+suppressPackageStartupMessages(c("tidyverse", "dplyr"))
```

Comment:
Since `dplyr` is already called by `library()` in the line right above this, it will print its package load messages. But turns out `dplyr` is actually loaded automatically with` tidyverse` so you can delete both mentions of `dplyr` here!